aspnet/mvc/views/forms/sample/final/Views/Demo/RegisterValidation.cshtml

```diff
@@ -0,0 +1,16 @@
+﻿@model RegisterViewModel
+
+<form asp-controller="Demo" asp-action="RegisterValidation" method="post">
+    <div asp-validation-summary="ValidationSummary.ModelOnly"></div>
+    Email:  <input asp-for="Email" /> <br>
+    <span asp-validation-for="Email"></span><br>
+    Password: <input asp-for="Password" /><br>
+    <span asp-validation-for="Password"></span><br>
+    <button type="submit">Register</button>
+</form>
+
+<script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-2.1.4.min.js"></script>
+<script src="https://ajax.aspnetcdn.com/ajax/jquery.validate/1.14.0/jquery.validate.min.js">
+</script>
+<script src="https://ajax.aspnetcdn.com/ajax/mvc/5.2.3/jquery.validate.unobtrusive.min.js">
```

Comment:
Well, the templates often lead people the wrong way. I was slightly incorrect: Browsers only warn when viewing a page w/ an HTTPS URL that contains HTTP links, not the other way round. But, regardless, there's no reason to use HTTPS for these links when the page is viewed using HTTP.

Exactly how are you getting a mixed content warning? Browsers should interpret `//ajax.aspnetcdn.com/ajax/mvc/5.2.3/jquery.validate.unobtrusive.min.js` as https://ajax.aspnetcdn.com/ajax/mvc/5.2.3/jquery.validate.unobtrusive.min.js when part of an HTTPS page.